.env.test

```diff
@@ -2,3 +2,5 @@
 # Override locally with `.env.test.local`
 
 SECRET_TOKEN="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+SITE_URL="test.host"
```

Comment:
Why don't we use localhost again? That would allow us to follow links in activation emails, for example. Or maybe it would enable asset loading in emails?

Reply:
:man_shrugging:

Reply:
Lets give it a try.

Reply:
Hmm... turns out a bunch of Spree routes in the test suite blow up if we change it. I can't immediately see how to fix it either. I'll revert this for now...